docs/www/configuration.md

```diff
@@ -678,7 +678,7 @@ Here is a more comprehensive view of the configration so that you can see all of
 | `dashboard_dir` | serve http dashboard on / url | None |
 | `default_num_windows` | Default number of quota tracking windows | 10 |
 | `default_topic_partitions` | Default number of partitions per topic | 1 |
-| `default_topic_replication` | Default replication factor for new topics | 1 |
+| `default_topic_replication` | Default replication factor for new topics | 1 (In v21.8.1 and higher the default is 3 for clusters with at least 3 nodes) |
```

Comment:
This should be actually default_topic_replications not default_topic_replication.